src/Typography/Caption/Caption.component.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import 'typeface-roboto';
```

Comment:
We don't want to bundle this font with each of these components. It would be better to let the user import fonts to the project however they want. So it's available in Storybook, we can move the Roboto import to `.storybook/config.js`.